ts/webui/src/components/trial-detail/DefaultMetricPoint.tsx

```diff
@@ -57,7 +58,13 @@ class DefaultPoint extends React.Component<DefaultPointProps, DefaultPointState>
         }
     };
 
-    generateGraphConfig(maxSequenceId: number): any {
+    pointClick = (params: any): void => {
+        if (window.location.pathname === '/oview') {
```

Comment:
Why this `if` is necessary?

Reply:
> current only support this operation on overview page (for top trials)

`<DefaultMetricPoint.tsx>` is used in detail page and overview page.